### ISSUE TITLE
High Priority - WCAG Level A - Button purpose, not possible to determine function or relation

### ISSUE THREAD

**Issue by JuliaSang** [w3c/w3c-website#165]
It is not possible to determine either the function of the button, nor what it relates to from the current labelling provided.

WCAG Reference:
4.1.2 Name, Role, Value (Level A)
[Understanding Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) | [How to Meet Name, Role, Value](https://www.w3.org/WAI/WCAG21/quickref/#name-role-value)

**Issue ID: DAC_Button_purpose_01**

URL: https://beta.w3.org/membership/list/ 
Page title: Current Members & Testimonials 

Journey 2, Step 1

<img width="904" alt="button" src="https://user-images.githubusercontent.com/20167528/159254825-5af9eaf7-80f5-47fc-bbb7-3d25c5124880.png">

The labels for the chip buttons to remove the filter options are insufficiently descriptive to indicate their purpose and what they relate to, announcing to screen reader users only as 'United Kingdom' for example.

This does not provide screen reader users with the buttons function, which is to remove a selected filter. Visually this is indicated by the styling of the buttons; but the same information is not provided programmatically.

Additional context should be provided in the button labelling to ensure that screen reader users are able to determine the purpose/ function of the button is clear; this is particularly important where more than one filter is selected. This information may be visually concealed to preserve the appearance of the page.

**Current Code Ref(s):** #countries-input-remove-206
 <button class="remove-option" type="button" id="countries-input-remove-206" aria- describedby="countries-input-remove" data-value="GB">United Kingdom </button>

**Solution:**
To ensure that the purpose of the button is clear for users who rely on audio feedback to navigate, it is recommended to include additional information within the labelling. This information can be included by way of a visually hidden <span>.

Additionally, the button has been provided with an aria-describedby attribute which does not reference a corresponding id; this attribute should be removed.

**Example:**
```
 <button class="remove-option" type="button" id="countries-input-remove-206 data-value="GB"><span class="sr-only"> Remove </span>United Kingdom </button>
 ```
**CSS:**
 ```
.sr-only
  position:    absolute !important;
  overflow:    hidden   !important;
  white-space: nowrap   !important;
  width:   1px !important;
  height:  1px !important;
  margin: -1px !important;
  padding: 0   !important;
  border:  0   !important;
  clip: rect(1px, 1px, 1px, 1px) !important;
  -webkit-clip-path: inset(50%) !important;
  clip-path: inset(50%) !important;
}
 ```
